packages/opentelemetry-node/docs/get-started.asciidoc

```diff
@@ -0,0 +1,209 @@
+////
+Goal of this doc:
+The user is able to successfully see data from their Node application make it to the Elastic UI via the Elastic Distribution for OpenTelemetry Node.js
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+* They have Node and NPM installed
+////
+
+[[get-started]]
+== Get started
+
+:language: Node.js
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+This guide shows you how to use the Elastic Distribution for OpenTelemetry {language} ("the distro") to instrument your {language} application and send OpenTelemetry data to an Elastic Observability deployment.
+
+[NOTE]
+====
+As an OpenTelemetry SDK, the distro supports sending data to any OpenTelemetry protocol (OTLP) endpoint (https://opentelemetry.io/docs/collector/[OpenTelemetry Collector]), but this guide assumes you are sending data to Elastic.
+====
+
+////
+✅ What the user needs to know and/or do before they install the distro
+////
+[discrete]
+[[get-started-prerequisites]]
+=== Prerequisites
+
+Before getting started:
+
+// Familiarity with OpenTelemetry
+* *Familiarize yourself with the OpenTelemetry SDK.* The distro is a lightweight wrapper around the OpenTelemetry SDK. You should be broadly familiar with the OpenTelemetry SDK concepts and instrumenting applications using the Microsoft diagnostic APIs. If you are not familiar with this, start with the https://opentelemetry.io/docs/languages/net[OpenTelemetry SDK documentation].
++
+// Any dependencies (including supported versions)
+* *Check your Node.js version*.
+include::supported-technologies.asciidoc[tag=node-version]
```

Comment:
>If we want to make the get-started more pithy

It's definitely a balancing act between pithy (to not overwhelm) and complete (to prevent frustration). I'm ok removing this point if they will get a clear error/warning message at the point of installation.